collect_app/src/main/java/org/odk/collect/android/utilities/ImageConverter.java

```diff
@@ -39,9 +39,26 @@
     private ImageConverter() {
     }
 
+    /**
+     * Before proceed with scaling or rotating, make sure existing exif information is store/restored.
+     * @author Khuong Ninh (khuong.ninh@it-development.com)
+     */
     public static void execute(String imagePath, QuestionWidget questionWidget, Context context) {
+        ExifInterface exif = null;
+        try {
+            exif = new ExifInterface(imagePath);
+        } catch (IOException e) {
+            Timber.w(e);
+        }
+
         rotateImageIfNeeded(imagePath);
         scaleDownImageIfNeeded(imagePath, questionWidget, context);
+        
+        try {
+            exif.saveAttributes();
```

Comment:
This might cause NPE. Please add a null check here.

Reply:
Thank you, @grzesiek2010 ,

I have updated the code to avoid NPE as you suggested.
Please kindly review